.github/workflows/sync-package-periodically.yml

```diff
@@ -29,18 +28,36 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: ${{ runner.os }}-node-
 
+      - name: Create new branch for changes
+        run: |
+            new_branch="update-metadata-${{ github.run_id }}"
+            git checkout -b "$new_branch"
+
       - name: Sync Package metadata
         run: scripts/syncMetadata.sh
 
       - name: Rebuild and validate website
         run: scripts/rebuild.sh
 
-      - name: Commit Results
+      - name: Commit and Push changes
         run: |
             git config --local user.email "actions@github.com"
             git config --local user.name "GitHub Actions"
-            git add scripts/commit.txt
-            git commit -m 'Data Updated' ||  echo 'No changes to commit'
-            git fetch origin main
-            git push origin HEAD:main || echo "Error occurred while push to origin"
-            
+            git add .
+            git commit -m 'Update package metadata and website content' || echo 'No changes to commit'
+            git push origin HEAD  || echo "Error occurred while push to origin"
```

Comment:
This almost certainly seems wrong since HEAD is not the branch trying to be pushed here.

Reply:
This actually does infer the name of the branch to push correctly (so, it should create `$new_branch`)